concepts/error-handling/links.json

```diff
@@ -1 +1,14 @@
-[]
+[
+    {
+        "url": "https://book.cairo-lang.org/ch09-00-error-handling.html",
+        "description": "Error handling in the Cairo book"
+    },
+    {
+        "url": "https://book.cairo-lang.org/ch09-01-unrecoverable-errors-with-panic.html#unrecoverable-errors-with-panic",
```

Comment:
The 2nd and 3rd URLs are redundant as they point to the same chapter in The Cairo Book, you can remove them